src/Spectre.Data.Tests/RoiUtilitiesTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿/*
+ * RoiUtilitesTests
+ * Class with tests for RoiUtilities class.
+
+   Copyright 2017 Roman Lisak
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework;
+using Spectre.Data.Datasets;
+using Spectre.Data.RoiIo;
+
+namespace Spectre.Data.Tests
+{
+    [TestFixture]
+    public class RoiUtilitiesTests
+    {
+        private readonly string _path = TestContext.CurrentContext.TestDirectory + "\\..\\..\\..\\..\\..\\test_files\\Rois";
+        private string _testpath;
+        private string _testfilespath;
+        private readonly RoiDataset _readroidataset = new RoiDataset();
+        private readonly RoiDataset _writeroidataset = new RoiDataset();
```

Comment:
My OCD is tingling when it comes to name variables in C'ish way in C# language or similar.
I don't know if we should take it as a matter of preference, I personally would refactor those names from `_nameofvariable` to `_nameOfVariable` like in every source file I stumbled upon in this project.